src/Automation/Automation/ChangeLog.md

```diff
@@ -21,6 +21,7 @@
 * Update help for Import-AzAutomationDscNodeConfiguration
 * Added configuration name validation to Import-AzAutomationDscConfiguration cmdlet
 * Improved error handling for Import-AzAutomationDscConfiguration cmdlet
+* Fixed issue when retreiving certain monthly schedules in several Azure Automation cmdlets
```

Comment:
Please pull from upstream master (git pull upstream master) and make sure that this new changelog entry is still under "Upcoming release"